common/source/docs/common-camera-gimbal.rst

```diff
@@ -177,25 +177,25 @@ See :ref:`Camera Shutter Configuration in Mission Planner <common-camera-shutter
 Camera Mount Mode/Targeting
 ===========================
 
-The camera/gimbal direction can be controlled by the pilot using RC control(RC Targeting) if RC channels for control have been assigned (default on startup unless changed), by the autopilot during missions using the DO_SET_ROI or DO_MNT_CONTROL commands (GPS and MAVLink Targeting), not at all (just stabilizing and set to a given angle on the axes, called NEUTRAL), or when RETRACTED if a retractable mount is used to rotate the camera as it retracts for clearance.
+The camera/gimbal direction can be controlled by the pilot using RC control(RC Targeting) if RC channels for control have been assigned (default on startup unless changed), by the autopilot during missions using the DO_SET_ROI or DO_MNT1_CONTROL commands (GPS and MAVLink Targeting), not at all (just stabilizing and set to a given angle on the axes, called NEUTRAL), or when RETRACTED if a retractable mount is used to rotate the camera as it retracts for clearance.
```

Comment:
DO_MNT1_CONTROL should be DO_MOUNT_CONTROL but this is only a decent choice for 4.2 (and earlier).